OrbitCore/OrbitModule.cpp

```diff
@@ -58,11 +78,10 @@ bool Module::LoadDebugInfo() {
   m_Pdb->SetMainModule(m_AddressStart);
 
   PRINT_VAR(m_FoundPdb);
-  if (m_FoundPdb) {
-    return m_Pdb->LoadDataFromPdb();
-  }
+  if (!m_FoundPdb) return false;
```

Comment:
nit: {}

Reply:
I actually do like this shorter style of having it in 1 line instead of 3. It feels more like a cohesive statement to me that can be quickly read and understood. I am very much open for a discussion about it. 

Google C++ Style Guide relevant section: 
Short conditional statements may be written on one line if this enhances readability. You may use this only when the line is brief and the statement does not use the else clause.

if (x == kFoo) return new Foo();
if (x == kBar) return new Bar();